### ISSUE TITLE
No named exports found in module refer to mjs file

### ISSUE THREAD

**Issue by lwr** [import-js/eslint-plugin-import#2228]
ref #2093 https://github.com/standard/standard/issues/863

```js
// File a.mjs
export const A = 'a'

// File b.js
export * from './a.mjs'

// File c.js
export * from './b.js'
```

```bash
eslint c.js
# output
#  1:15  error  No named exports found in module './b.js'  import/export

eslint c.js --ext .js,.mjs
# output
#  1:15  error  No named exports found in module './b.js'  import/export

```

only renaming `a.mjs` to `a.js` can resolve this problem

**Comment by ljharb** [comment]
Do you have `.mjs` in the `import/extensions` setting?

**Comment by lwr** [comment]
> Do you have `.mjs` in the `import/extensions` setting? That seemed to be the fix for #2093.

mjs is the official extension for modules in NodeJS, so I think that it should be parsed by default without any addition config

**Comment by ljharb** [comment]
That's true, but it can't happen until `resolve` supports "exports".

**Comment by Derranion** [comment]
I can confirm that this issue is still exists - got it recently with `.ts` files.

Here is another good [example](https://stackoverflow.com/questions/55035818/how-to-fix-disable-seemingly-erroneous-no-named-exports-found-in-module-error).

Would love to see it fixed because `import \ export` check is pretty important.
( as a workaround for now you could change rule to throw `warning` instead of `error` )

Thanks a lot.
